doc/python/images.md

```diff
@@ -305,5 +305,60 @@ fig.update_layout(
 fig.show(config={'doubleClick': 'reset'})
 ```
 
+### Annotating layout image with shapes
+
+_introduced in plotly 4.7_
+
+It can be useful to add shapes to a layout image, for highlighting an object, drawing bounding boxes as part of a machine learning training set, or identifying seeds for a segmentation algorithm. 
+
+In order to enable shape drawing, you need to
+- define a dragmode corresponding to a drawing tool (`'drawline'`,`'drawopenpath'`, `'drawclosedpath'`, `'drawcircle'`, or `'drawrect'`)
+- add modebar buttons corresponding to the drawing tools you wish to use.
+
+The style of new shapes is specified by the `newshape` layout attribute. Shapes can be selected and modified after they have been drawn. More details and examples are given in the [tutorial on shapes](/python/shapes#drawing-shapes-on-cartesian-plots).
+
+Drawing or modifying a shape triggers a `relayout` event, which [can be captured by a callback inside a Dash application](https://dash.plotly.com/interactive-graphing). 
```

Comment:
Now it triggers `_guiRelayout` instead of `relayout`.

Reply:
thanks I'll change this. For a Dash app, does `_guiRelayout` fall in the realm of `relayoutData` events? I tried with the bundle from yesterday afternoon and my Dash apps still work as expected with callbacks listening to `relayoutData`.

Reply:
I think we don't need to tell Python users the name of the actual underlying event, just how to access it from widgets and Dash.

Reply:
OK. Resolving...